sdks/java/io/hcatalog/resources/testconf/hive-log4j2.properties

```diff
@@ -0,0 +1,123 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Is all of this configuration really necessary to bring up an in-memory instance of Hive for unit testing?